sycl/test-e2e/ClusterLaunch/cluster_launch_enqueue_functions.cpp

```diff
@@ -0,0 +1,59 @@
+// REQUIRES: cuda
+// RUN: %{build} -Xsycl-target-backend --cuda-gpu-arch=sm_90 -o %t.out
+// RUN: %{run} %t.out
+
+#include <sycl/ext/oneapi/experimental/cluster_group_prop.hpp>
+#include <sycl/ext/oneapi/experimental/enqueue_functions.hpp>
+#include <sycl/sycl.hpp>
+
+#include <string>
+
+int main() {
+  using namespace sycl::ext::oneapi::experimental;
+
+  sycl::queue queue;
+  auto computeCapability = std::stof(
+      queue.get_device().get_info<sycl::info::device::backend_version>());
+
+  if (computeCapability >= 9.0) {
+    cuda::cluster_size cluster_dims(sycl::range<3>(2, 2, 1));
+    properties cluster_launch_property{cluster_dims};
+
+    int *correct_result_flag = sycl::malloc_device<int>(1, queue);
+    queue.memset(correct_result_flag, 1, sizeof(int)).wait();
```

Comment:
```suggestion
    queue.memset(correct_result_flag, 0, sizeof(int)).wait();
```
Set this to zero and only set it to 1 if correct